gammapy/analysis/config.py

```diff
@@ -130,16 +143,19 @@ class BackgroundConfig(GammapyBaseConfig):
     method: BackgroundMethodEnum = BackgroundMethodEnum.reflected
     exclusion: FilePath = None
 
+class SafeMaskSettingsConfig(GammapyBaseConfig):
+    offset_max: AngleType = "2.5 deg"
+    aeff_percent: float = 10
+    bias_percent: float = 10
```

Comment:
I guess these are not the whole collection of parameters needed for all the possible safe mask methods. Also note that you could have `None` values by default for all these other params missing.

Reply:
Yes I think we miss a `position` parameter that would take a `SkyCoord` in the parameter list (I am not sure if we would really use that option, but it exists at the moment). And this could also be adapted, as requested in #2704.